node/index.ts

```diff
@@ -63,6 +61,13 @@ export default {
     robots,
     sitemap,
     user: userSitemap,
+
+
```

Comment:
A single blank line is enough, isn't it?

Reply:
no, tslint still complains

Reply:
Weird, works for me.